tests/testing.py

```diff
@@ -50,10 +50,13 @@ class UbuntuAdvantageTest(TestWithFixtures):
     def setUp(self):
         super(UbuntuAdvantageTest, self).setUp()
         self.tempdir = self.useFixture(TempDir())
-        self.repo_list = Path(self.tempdir.join('repo.list'))
         self.boot_cfg = Path(self.tempdir.join('boot.cfg'))
         self.fstab = Path(self.tempdir.join('fstab'))
         self.cpuinfo = Path(self.tempdir.join('cpuinfo'))
+        self.esm_repo_list = Path(self.tempdir.join('esm-repo.list'))
+        self.fips_repo_list = Path(self.tempdir.join('esm-repo.list'))
```

Comment:
Should this be self.tempdir.join('fips-repo.list')?

Reply:
good catch, fixed